source/includes/_targets.md

```diff
@@ -0,0 +1,384 @@
+# Targets
+
+A Target resource is a sub-resource related to an Advocacy Campaign.
+Each Advocacy Campaign contains an array of targets.  Each Target contains the following attributes.
+
+Name    | Type | Description
+--------- |  ----------- |  -----------
+identifiers | array | A unique string array of identifiers in the format [system name]:[id]. See the general concepts document for more information about identifiers.
+created_at | datetime | A read-only property representing the date and time the resource was created on the local system.
+updated_at | datetime |A read-only property representing the date and time the resource was last modified on the local system.
+title | string | The title or position of the target. (ex: “Senator” or “CEO”)
+organization | string |The organization the target belongs to. (ex: “U.S. Senate” or “Acme Corporation”)
+given_name | string | The first or given name of the target. (ex: “John”)
+family_name | string | The last or family name of the target. (ex: “Smith”)
+ocdid | string | array | The Open Civic Data Division ID for this target’s political geography, if applicable. See here for more documentation. (ex: “ocd-division/country:us/state:ny/cd:18”, which corresponds to New York’s 18th Congressional District)
```

Comment:
"See here for more documentation" should be hotlinked to whatever it's supposed to refer to